.dockerignore

```diff
@@ -54,6 +54,5 @@ out*
 .byebug_history
 .gitignore
 .dockerignore
-.travis.yml
```

Comment:
Can you add `.github/` instead? seems to be missing here

Reply:
sure, done :+1: